babi/file.py

```diff
@@ -377,7 +377,15 @@ def ctrl_right(self, dim: Dim) -> None:
         else:
             self.buf.right(dim)
             tp = line[self.buf.x].isalnum()
-            while self.buf.x < len(line) and tp == line[self.buf.x].isalnum():
+            while (
+                self.buf.x < len(line) and
+                tp == line[self.buf.x].isalnum() and
+                not (
+                    self.buf.x == 1 and
```

Comment:
I don't think this is sufficient -- the same problem would happen halfway through an indent if I understand correctly

Reply:
Do you think of case when we have \t\tSome words
And when we are on the 2nd tab we will jump to the end of "Some" word. I done it the same as spaces works, if you are on the space before the word you jump at the end of the word, and that is included in test. If you think behaviour should be different i can think of solution. I do not want to differentitate behaviour for space  and space like chcaracters  ( tab ) included.

Reply:
or cursor is at position 3 of a 4 space tab

Reply:
Ok so you have in maind case when we have characters in intedn from tab? I still do not know what is expected behaviour from you. 
You want to treat \t as 4 spaces and it should work the same? so when we have situatuation like this:
`sad     indent
     ^   ^        ^
      |    |          |    + when use tab as work separator
      |    |    + when spaces as word separator
      |    + cursor postion
`
Do you want to override default behavior and when we have words separated by tab we should go to the start od the word?

Reply:
Ok. I think I understand what you are thinking off. I will try to solve that :)

Reply:
I used the constatnt 3 and 4 and probably i should use some config variable which tells the width of tab. If you can tell me if direction i take is ok and what variable i should look for ?

Reply:
yeah that doesn't look like the right direction at all.  I suspect what actually should be done is to find the "break" points in the line and navigate to the next one

Reply:
the width of the tab doesn't matter

Reply:
Ok. So you want it working like in nano? So when tab is used it is alweys go to the start of the next word?

Reply:
Ok. I make it the most trivial now how it can be. I do not really get the better idea of how to find break points without knowing the tab lenght. Current solution resolve most cases except one when we have something like that:
asd\tasd -> it will go to the a of the asd instead of the end. But i do not really know how i should find if we should go after the word or not without knowing the tab size and counting characters in the string.